hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceImpl.java

```diff
@@ -102,45 +102,38 @@
     @SuppressWarnings("checkstyle:visibilitymodifier")
     public final Node node;
 
-    final ILogger logger;
+    final ConcurrentMap<String, Object> userContext = new ConcurrentHashMap<String, Object>();
 
+    final ILogger logger;
     final String name;
-
     final ManagementService managementService;
-
     final LifecycleServiceImpl lifecycleService;
-
     final ManagedContext managedContext;
-
-    final ConcurrentMap<String, Object> userContext = new ConcurrentHashMap<String, Object>();
-
     final HazelcastInstanceCacheManager hazelcastCacheManager;
 
     @SuppressWarnings("checkstyle:executablestatementcount")
-    protected HazelcastInstanceImpl(String name, Config config, NodeContext nodeContext)
-            throws Exception {
+    protected HazelcastInstanceImpl(String name, Config config, NodeContext nodeContext) {
         this.name = name;
         this.lifecycleService = new LifecycleServiceImpl(this);
 
         ManagedContext configuredManagedContext = config.getManagedContext();
-        managedContext = new HazelcastManagedContext(this, configuredManagedContext);
+        this.managedContext = new HazelcastManagedContext(this, configuredManagedContext);
 
-        //we are going to copy the user-context map of the Config so that each HazelcastInstance will get its own
-        //user-context map instance instead of having a shared map instance. So changes made to the user-context map
-        //in one HazelcastInstance will not reflect on other the user-context of other HazelcastInstances.
-        userContext.putAll(config.getUserContext());
-        node = createNode(config, nodeContext);
+        // we are going to copy the user-context map of the Config so that each HazelcastInstance will get its own
+        // user-context map instance instead of having a shared map instance. So changes made to the user-context map
+        // in one HazelcastInstance will not reflect on other the user-context of other HazelcastInstances
+        this.userContext.putAll(config.getUserContext());
```

Comment:
What is the added value of adding 'this'? There is no ambiguity; so it is obvious that this is a field. Adding 'this' just add horizontal noise.

Reply:
I think the additional `this.` can actually improve the readability:
```java
ConstructorExample(Object foo, Object bar) {
    this.foo = foo;
    this.baz = initBaz();
    this.bar = bar;
}
```
vs.
```java
ConstructorExample(Object foo, Object bar) {
    this.foo = foo;
    baz = initBaz();
    this.bar = bar;
}
```
For me the fully aligned version is much easier to read. It's also more consistent to use `this.` for all fields in a constructor, so it's easier to distinct between fields and local variables (in addition to the color highlighting of the IDE).